src/moveFile.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-console */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const moveFile = (oldPath, newPath) => {
+  const destPath = fs.existsSync(newPath)
```

Comment:
The check for existence of newPath is not accurate. The newPath could be an existing file, not necessarily a directory.